_posts/2020-03-30-Node-modem-taking-the-plunge.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Node modem is taking the plunge
+banner : images/banner-pulse-standard.jpg
+date:  2020-03-30
+categories: wnc
+thumbnail: images/pulse-thumbnail-node-modem.jpeg
+---
+Our [Silver edition](https://subnero.com/products/silver.html#title) (Node Configuration) modem is taking the plunge in the Mediterranean Sea.
+
+The team of Dr Roee Diamant, at The University of Haifa, deployed a Node configuration Subnero modem with an additional [co-processor](https://subnero.com/wnc/2018/11/17/Underwater-modem-with-a-coprocessor.html), that is configured with sonar-like functionalities, for their research.
```

Comment:
Suggest adding a link to his name:  [Dr. Roee Diamant](https://sites.google.com/edu.haifa.ac.il/anl/staff)

Reply:
Good idea. Done. :-)